javascript/javascript2/week2/workingWithMovies.js

```diff
@@ -0,0 +1,28 @@
+//Create an array of movies containing the movies with a short title
+const shortTitleMovies = movies.filter (movie => movie.title.length < 3)
+
+//Create an array of movie titles with long movie titles
+const longTitles = movies.filter (movie => movie.title.length > 50).map (title => title.title)
```

Comment:
You can use destructuring for these:
```js
const longTitles = movies.filter(({title}) => title.length > 50).map (({title}) => title) // you can do this because the object that's used in the filter and map has a "title" property
```

Reply:
nice!